app/js/bus/api/commonApiBase.js

```
@@ -184,6 +185,114 @@ ozpIwc.CommonApiBase.prototype.loadFromEndpoint=function(endpointName, requestHe
+			// if any embedded items exist, convert them to a list (primarily to cover the single element case
+			if (data.response._embedded && data.response._embedded.item) {
+				if (Object.prototype.toString.call(data.response._embedded.item) === '[object Array]' ) {
+					for (var i in data.response._embedded.item) {
+						embeddedList.push(data.response._embedded.item[i]);
+					}
+				} else {
+					embeddedList.push(data.response._embedded.item);
+				}
+				
+				embeddedList.forEach(function(item) {
+					self.updateResourceFromServerIterative(item, item._links.self.href, endpoint, requestHeaders);
+				});
+			}
+			
+			// Follow up here with loop on _links section, creating a promise to load each link if it is not in the _embedded section
+			// At end, return promise.all to activate when all outstanding loads are completed.
+			if (data.response._links && data.response._links.item) {
+				var links = [];
+				if (Object.prototype.toString.call(data.response._links.item) === '[object Array]' ) {
+					for (var i in data.response._links.item) {
+						links.push(data.response._links.item[i]);
+					}
+				} else {
+					links.push(data.response._links.item);
+				}
+				
+				// scan the list of links.  If there is no href match to an embedded item, push a promise to load the link into the list
+				links.forEach(function(link) {
+					if (!embeddedList.some(
+							function(embeddedItem) { return link.href === embeddedItem._links.self.href }
+						))
+						unresolvedLinks.push(endpoint.get(link.href, requestHeaders));
+				});
+			}
+			return Promise.all(unresolvedLinks);
+		}).then(function(unresolvedLinks) {
+			unresolvedLinks.forEach(function(item) {
+				self.updateResourceFromServerIterative(item, item._links.self.href, endpoint, requestHeaders);
+			});
+
+			// update all the collection values
+			self.dynamicNodes.forEach(function(resource) {
+				self.updateDynamicNode(self.data[resource]);
+			});
+		})['catch'](function(e) {
```

Remove this catch block.  It will eat the error and prevent the caller from implementing a `catch` of their own.